bugbot/crash/analyzer.py

```diff
@@ -18,24 +18,30 @@
 from bugbot.components import ComponentName
 from bugbot.crash import socorro_util
 
+# The max offset from a memory address to be considered "near".
+OFFSET_64_BIT = 0x1000
+OFFSET_32_BIT = 0x100
 # Allocator poison value addresses.
 ALLOCATOR_ADDRESSES_64_BIT = (
-    0xE5E5E5E5E5E5E5E5,
-    0x4B4B4B4B4B4B4B4B,
+    (0xE5E5E5E5E5E5E5E5, OFFSET_64_BIT),
+    # On 64-bit windows, sometimes it could be doing something with a 32-bit
+    # value gotten from freed memory, so it'll be 0X00000000E5E5E5E5 +/-, and
+    # because of the address limitation, quite often it will be
+    # 0X0000E5E5E5E5E5E5 +/-.
+    (0x00000000E5E5E5E5, OFFSET_32_BIT),
+    (0x0000E5E5E5E5E5E5, OFFSET_64_BIT),
+    (0x4B4B4B4B4B4B4B4B, OFFSET_64_BIT),
```

Comment:
@gabrielesvelto does this list of targeted addresses and their offset values appear correct to you?

Reply:
Yes, these are looking good. You can always validate them by looking at the crashes we have on file and how they exhibit those patterns. See:
* This [query](https://crash-stats.mozilla.org/search/?address=~4b4b&_facets=signature&_sort=-date&_columns=date&_columns=version&_columns=address#crash-reports) for the `4b` pattern
* And this [query](https://crash-stats.mozilla.org/search/?address=~e5e5&_facets=signature&_sort=-date&_columns=date&_columns=version&_columns=address#crash-reports) for the `e5` pattern

Reply:
Thank you @gabrielesvelto!